docker/docker-compose.yml

```diff
@@ -0,0 +1,29 @@
+name: cwydsa
+services:
+  web:
+    image: fruoccopublic.azurecr.io/rag-webapp
+    ports:
+      - "8080:80"
+    env_file:
+      - path: ../.env
+        required: false
+      - path: $AZD_ENV_FILE
+        required: false
+  admin:
+    image: fruoccopublic.azurecr.io/rag-adminwebapp
+    ports:
+      - "8081:80"
+    env_file:
+      - path: ../.env
+        required: false
+      - path: $AZD_ENV_FILE
+        required: false
+  backend:
```

Comment:
I'm not sure how useful this is, because in order to invoke the endpoints it requires the `FUNCTION_KEY` env var. Unless you could write a script to fetch the key from Azure and then apply that to the environment?

Reply:
And then for the admin app we would need to configure `BACKEND_URL` to point to the function running locally, within Docker

Reply:
![i got it](https://media.giphy.com/media/v1.Y2lkPTc5MGI3NjExYjhtdDB2Z2tlN2V5eG93MHkyaG9hdjMydGVnMnR4NXYwMWRoZ3NmOSZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/vRL71JP8OjhghxccYb/giphy.gif)

Reply:
Fair play haha

![image](https://media.giphy.com/media/v1.Y2lkPTc5MGI3NjExYXNldHJ0eGF6dTJ0aWl5bGJyamRrb2U1MGl3MDhneHFseHY5YTB4YyZlcD12MV9naWZzX3NlYXJjaCZjdD1n/YGJBp5EgyVP9K/giphy.gif)